src/meta/user/user.ts

```diff
@@ -6,15 +6,23 @@ export enum UserStatus {
   inactive = 'inactive',
 }
 
+export type UserProps = {
+  // old fields
+  name: string
+  lang: string
+  institution?: string
+  position?: string
```

Comment:
to remove. in migration they will become role props (organization, position)

Reply:
ok, so they will be replicated for every role of the user. but position will be Professional title / Titre professionnelle / Título professional

Reply:
next pr